<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorGradientWidgetClass</class>
 <widget class="QWidget" name="ColorGradientWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="num_Color">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Color Preview</string>
   </property>
   <widget class="ColorGradientPreview" name="PreviewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_confirm">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbField">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Start Color</string>
   </property>
   <widget class="QPushButton" name="Bt_BeginColor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>D:/GIS_todo/Dev20图标 (1)/Dev20/ColorMixer_32x32.png</normaloff>D:/GIS_todo/Dev20图标 (1)/Dev20/ColorMixer_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="ColorWidget" name="begincolor_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>End Color</string>
   </property>
   <widget class="ColorWidget" name="endcolor_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_EndColor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>D:/GIS_todo/Dev20图标 (1)/Dev20/ColorMixer_32x32.png</normaloff>D:/GIS_todo/Dev20图标 (1)/Dev20/ColorMixer_32x32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set the number of sorts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the attribute:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorGradientPreview</class>
   <extends>QWidget</extends>
   <header location="global">ColorGradientPreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header location="global">ColorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
